iso20022-core/src/main/java/com/prowidesoftware/swift/model/SettlementInfo.java

```diff
@@ -0,0 +1,42 @@
+package com.prowidesoftware.swift.model;
+
+/**
+ * Class for identification of MX messages.
+ *
+ * <p>It is composed of the Settlement Method and Clearing System (Code or Property).
+ * Used in combination with MxId to identify a specific MX message.
+ *
+ * @since 9.5.5
+ */
+public class SettlementInfo {
+
+    private SettlementMethod settlementMethod;
+    private String clrSysCd; // Clearing System Code
+    private String clrSysPrtry; // Clearing System Proprietary Code
```

Comment:
rename it to clearingSystemCode and as a single field, we do not care if it comes from code or proprietary